cpp/src/parquet/file_writer.cc

```diff
@@ -307,7 +321,17 @@ class RowGroupSerializer : public RowGroupWriter::Contents {
       auto codec_options = properties_->codec_options(path)
                                ? (properties_->codec_options(path)).get()
                                : nullptr;
-
+      BloomFilter* bloom_filter = nullptr;
```

Comment:
These lines duplicate with line 162 above, can we consolidate them into `GetOrCreateBloomFilter` call?

Reply:
Done, it should be better!